server/plugin/pipeline.go

```diff
@@ -222,20 +227,25 @@ func (p *Plugin) executePipelineGetWorkflowByID(args *model.CommandArgs,
 }
 
 func (p *Plugin) executeTriggerPipeline(args *model.CommandArgs,
-	token string, projectSlug string) (*model.CommandResponse, *model.AppError) {
+	token string, projectSlug string, branch string) (*model.CommandResponse, *model.AppError) {
 	p.API.LogError("trigger pipeline called")
-	pl, err := circle.TriggerPipeline(token, projectSlug)
+	pl, err := circle.TriggerPipeline(token, projectSlug, branch)
 	if err != nil {
+		p.API.LogError("uparmar", err.Error())
```

Comment:
What does "uparmar" means?

Also, `err.Error()` should have a key with it:
```suggestion
		p.API.LogError("uparmar", "err", err.Error())
```

Reply:
ohh i added it for debugging purposes and uparmar is to make my logs stand out from other logs! will remove it.